QA_Requests/PerfTickets/testPages/subcase_query/import_generated_data.py

```diff
@@ -0,0 +1,51 @@
+import time
+
+from selenium.webdriver.common.by import By
+from QA_Requests.PerfTickets.testPages.base.base_page import BasePage
+from QA_Requests.PerfTickets.userInputs.test_data import TestData
+
+
+class DataGenerationPage(BasePage):
```

Comment:
would it not be easier to use the bulk import API: https://confluence.dimagi.com/display/commcarepublic/Bulk+Upload+Case+Data

Reply:
Wow, I didn't know this existed - I'm sure this will take significantly less time than this method.
Thanks for pointing this out @snopoke -- Going to try it out next time.

Reply:
I removed UI to import and added this instead https://github.com/dimagi/dimagi-qa/blob/2777da2b8d1508adc59bd5cfb8427ae800f40b68/QA_Requests/PerfTickets/testPages/subcase_query/import_generated_data.py 
Does this look fine @snopoke ?